Makefile

```
@@ -1,157 +1,74 @@
+
+.PHONY: all build release clean update format vet test tidyup container githubActions gifs
 
 all: build
-check: format vet test
 
-build: 	clean update tidyup format vet test \
-		create_dirs build_linux_amd64 build_linux_static build_linux_arm \
-		build_windows build_macos build_macos_arm build_freebsd
-	@echo
-	@echo "[+] Done"
-
-create_dirs:
-	@echo
-	@echo "[+] Version: $(VERSION)"
-	@echo
-	@mkdir -p $(EXEC_DIR)
-	@mkdir -p $(DEB_PACKAGE_DIR)
-	@mkdir -p $(DEBIAN_DIR)
-	@mkdir -p $(TARGET_EXECUTABLE_PATH)
-
-build_linux_amd64:
-	@echo "[+] Building the Linux version"
-	@go build -ldflags "-s -w" -o $(EXEC_DIR)tcping
-
-	@echo "[+] Packaging the Linux version"
-	@tar -czvf $(EXEC_DIR)tcping_Linux.tar.gz -C $(EXEC_DIR) tcping > /dev/null
-	@sha256sum $(EXEC_DIR)tcping_Linux.tar.gz
-
-	@echo "[+] Removing the Linux binary"
-	@rm $(EXEC_DIR)tcping
-
-
-build_linux_static:
-	@echo
-	@echo "[+] Building the static Linux version"
-	@env GOOS=linux CGO_ENABLED=0 go build -ldflags "-s -w" -o $(EXEC_DIR)tcping
-
-	@echo "[+] Packaging the static Linux version"
-	@tar -czvf $(EXEC_DIR)tcping_Linux_static.tar.gz -C $(EXEC_DIR) tcping > /dev/null
-	@sha256sum $(EXEC_DIR)tcping_Linux_static.tar.gz
-
-	@$(MAKE) build_debian_package
-
-	@echo "[+] Removing the static Linux binary"
-	@rm $(EXEC_DIR)tcping
-
-build_debian_package:
-	@echo
-	@echo "[+] Building the Debian package"
-	@cp $(EXECUTABLE_PATH) $(TARGET_EXECUTABLE_PATH)
-
-	@echo "[+] Creating the Debian control file"
-	@echo "Package: $(PACKAGE_NAME)" > $(CONTROL_FILE)
-	@echo "Version: $(VERSION)" >> $(CONTROL_FILE)
-	@echo "Section: custom" >> $(CONTROL_FILE)
-	@echo "Priority: optional" >> $(CONTROL_FILE)
-	@echo "Architecture: amd64" >> $(CONTROL_FILE)
-	@echo "Essential: no" >> $(CONTROL_FILE)
-	@echo "Maintainer: $(MAINTAINER)" >> $(CONTROL_FILE)
-	@echo "Description: $(DESCRIPTION)" >> $(CONTROL_FILE)
-
-	@echo "[+] Running dpkg-deb build"
-	@dpkg-deb --build $(DEB_PACKAGE_DIR)
-
-	@echo "[+] Renaming the Debian package"
-	@mv $(DEB_PACKAGE_DIR).deb $(EXEC_DIR)/$(PACKAGE)
-
-build_linux_arm:
-	@echo
-	@echo "[+] Building the Linux ARM version"
-	@env GOOS=linux GOARCH=arm64 go build -ldflags "-s -w" -o $(EXEC_DIR)tcping
-
-	@echo "[+] Packaging the Linux ARM version"
-	@tar -czvf $(EXEC_DIR)tcping_Linux_ARM.tar.gz -C $(EXEC_DIR) tcping > /dev/null
-	@sha256sum $(EXEC_DIR)tcping_Linux_ARM.tar.gz
-
-	@echo "[+] Removing the Linux ARM binary"
-	@rm $(EXEC_DIR)tcping
-
-build_windows:
-	@echo
-	@echo "[+] Building the Windows version"
-	@env GOOS=windows GOARCH=amd64 go build -ldflags "-s -w" -o $(EXEC_DIR)tcping.exe
-
-	@echo "[+] Packaging the Windows version"
-	@zip -j $(EXEC_DIR)tcping_Windows.zip $(EXEC_DIR)tcping.exe > /dev/null
-	@sha256sum  $(EXEC_DIR)tcping_Windows.zip
+# Build for current platform
+build: $(TARGET_DIR)/$(BIN_NAME)
 
-	@echo "[+] Removing the Windows binary"
-	@rm $(EXEC_DIR)tcping.exe
+# Build all release artifacts
+release: $(RELEASE_ARTIFACTS)
 
```

The CI fails without this:

```suggestion

check: format vet test

```

---

I might have messed the formatting here. Did it through my phone